tests/test_suite/unittests/test_find_modules.py

```diff
@@ -42,3 +43,28 @@ def test_finding_modules_py_dot_py_using_slashes_with_full_filename(data_dir):
     expected = sorted(data_dir / Path(*path) for path in paths)
     results = sorted(find_modules(data_dir / 'a' / 'py.py'))
     assert expected == results
+
+
+def test_small_directory_tree_with_excluding_files(data_dir):
+    paths = (('a', 'b.py'), ('a', 'py.py'),
+             ('a', 'c', 'd.py'))
+    excluded_modules = ['**/__init__.py']
+    expected = set(Path(*path) for path in paths)
+
+    from cosmic_ray.execution.local import excursion
```

Comment:
Just import this once at the top of the file (and remove the import below as well).

Reply:
ok